### PR TITLE
Build nimbus-fml and cli with Ubuntu 20.04 [ci full]

### PR DESCRIPTION
**Pull request by brennie**
### Pull Request checklist ###

- **Breaking changes**:  This PR follows our [breaking change policy](https://github.com/mozilla/application-services/blob/main/docs/howtos/breaking-changes.md)
  - [x] This PR follows the breaking change policy:
     - This PR has no breaking API changes, or
     - There are corresponding PRs for our consumer applications that resolve the breaking changes and have been approved
- [x] **Quality**: This PR builds and tests run cleanly
  - Note:
    - For changes that need extra cross-platform testing, consider adding `[ci full]` to the PR title.
    - If this pull request includes a breaking change, consider [cutting a new release](https://github.com/mozilla/application-services/blob/main/docs/howtos/cut-a-new-release.md) after merging.
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes a changelog entry in [CHANGELOG.md](../CHANGELOG.md) or an explanation of why it does not need one
  - Any breaking changes to Swift or Kotlin binding APIs are noted explicitly
- [x] **Dependencies**: This PR follows our [dependency management guidelines](https://github.com/mozilla/application-services/blob/main/docs/dependency-management.md)
  - Any new dependencies are accompanied by a summary of the due dilligence applied in selecting them.

[Branch builds](https://github.com/mozilla/application-services/blob/main/docs/howtos/branch-builds.md): add `[firefox-android: branch-name]` to the PR title.

### CODE REVIEW

File: taskcluster/scripts/toolchain/build-rust-toolchain.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ rustup --version
 rustup target add x86_64-apple-darwin
 rustup target add x86_64-pc-windows-gnu
 rustup target add x86_64-unknown-linux-musl
+rustup target add aarch64-unknown-linux-gnu
 rustup target add aarch64-unknown-linux-musl
```
Comment (jhugman) [inline taskcluster/scripts/toolchain/build-rust-toolchain.sh:34]: Do we still support/need this?

File: taskcluster/docker/linux2004/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    pyyaml \
+    toml \
+    taskcluster
+
+# tooltool
+RUN \
+    curl -sfSL --retry 5 --retry-delay 10 \
+         -o /usr/local/bin/tooltool.py \
+         https://raw.githubusercontent.com/mozilla-releng/tooltool/master/client/tooltool.py && \
+         chmod +x /usr/local/bin/tooltool.py
```
Comment (jhugman) [inline taskcluster/docker/linux2004/Dockerfile:74]: Out of interest, what are we using `tooltool` for here?

**Review by jhugman — APPROVED** [review]
This looks really nice. You should also add a line to [the `nimbus-cli` installer][0]. I'm fairly sure the [`NimbusGradlePlugin.groovy`][1] will work right away.

[0]: https://github.com/mozilla/application-services/blob/main/install-nimbus-cli.sh#L87
[1]: https://github.com/mozilla/application-services/blob/008426d87f6c806d86b4a26450e9a70e179acebc/tools/nimbus-gradle-plugin/src/main/groovy/org/mozilla/appservices/tooling/nimbus/NimbusGradlePlugin.groovy#L225-L229

File: taskcluster/docker/linux2004/Dockerfile (hunk 2 above)
Comment (brennie) [reply taskcluster/docker/linux2004/Dockerfile:74]: `tooltool` is used to fetch artifacts from previous tasks. We need this to grab the docker image and the rust compiler.

File: taskcluster/scripts/toolchain/build-rust-toolchain.sh (hunk 1 above)
Comment (brennie) [reply taskcluster/scripts/toolchain/build-rust-toolchain.sh:34]: You added this in another patch, right? We can probably drop it if you're not going to use it. Once there are aarch64 native builders we should probably switch to compiling for a -musl target to be strictly more portable.

**Pull request merged**